### DISCUSSION TITLE
`winget` support on Windows?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [biomejs/biome#1439] [Ideas]
winget is a breeze to install packages on Windows. it is officially supported by microsoft.

why not release biome there?

currently, the installation is a bit difficult than using winget.

would love support for it.

**Comment by ematipico** [comment]
That's definitely in our interests, but we would need all the help from the community!

So if you're willing to set up everything (with our support obviously) and document it, that would be really appreciated!
